src/bundle/Resources/views/themes/admin/ui/field_type/edit/ezgmaplocation.html.twig

```diff
@@ -3,7 +3,7 @@
 {%- block ezplatform_fieldtype_ezgmaplocation_widget -%}
     <div class="ez-data-source__map"></div>
     <div class="ez-data-source__locate-me">
-        <div class="form-control-label">{{ 'content.field_type.ezgmaplocation.set_current_location'|trans|desc('Set current location') }}</div>
+        <div class="form-control-label">{{ 'content.field_type.ezgmaplocation.set_current_location'|trans|desc('Use current location') }}</div>
```

Comment:
```suggestion
        <div class="form-control-label">{{ 'content.field_type.ezgmaplocation.set_current_location'|trans|desc('Use current Location') }}</div>
```

Reply:
This is not "Location" as a domain term, it's the user's actual location, to be marked on the map